packages/api/src/command/medical/patient/get-patient.ts

```diff
@@ -61,62 +69,66 @@ export const getPatientIds = async ({
   return patients.map(p => p.id);
 };
 
+/**
+ * Retrieves a patient based on their demographic information. Utilizes functions
+ * imported from the MPI core module: normalization, finding(blocking), matching, merging
+ * @param facilityId - The ID of the facility where the patient is associated.
+ * @param cxId - The ID of the patient in the external system.
+ * @param demo - The demographic information of the patient.
+ * @returns The matched patient object if found, otherwise undefined.
+ */
 export const getPatientByDemo = async ({
-  facilityId,
   cxId,
   demo,
 }: {
-  facilityId: string;
   cxId: string;
   demo: PatientData;
-}): Promise<Patient | null> => {
-  const { log } = Util.out(`getPatientByDemo - cxId ${cxId}`);
+}): Promise<Patient | undefined> => {
+  // TODO remove this once we have a proper refactor of Patient, PatientData, PatientDemographics
+  const patient: Patient = {
+    cxId,
+    eTag: "",
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    facilityIds: [],
+    id: "",
+    data: demo,
+  };
 
```

Comment:
This is a consequence of using Patient in core, but not having a patient defined yet, only demographcis